unrpyc.py

```diff
@@ -187,64 +188,133 @@ def worker(args, filename):
+def set_cpu_num():
+    """Sets the number of used CPUs.
+    3/4 of available, 1 if cores below 3. Overall max is 8."""
+    num_cpu = cpu_count()
+    return round(num_cpu * 0.75) if 2 < num_cpu < 11 else 8 if num_cpu > 10 else 1
 
-    parser.add_argument('-l', '--language', dest='language', action='store', default='english',
-                        help="if writing a translation file, the language of the translations to write")
 
-    parser.add_argument('--comparable', dest='comparable', action='store_true',
-                        help="Only for dumping, remove several false differences when comparing dumps. "
-                        "This suppresses attributes that are different even when the code is identical, such as file modification times. ")
-
-    parser.add_argument('--no-pyexpr', dest='no_pyexpr', action='store_true',
-                        help="Only for dumping, disable special handling of PyExpr objects, instead printing them as strings. "
-                        "This is useful when comparing dumps from different versions of Ren'Py. "
-                        "It should only be used if necessary, since it will cause loss of information such as line numbers.")
-
-    parser.add_argument('--no-init-offset', dest='init_offset', action='store_false',
-                        help="By default, unrpyc attempt to guess when init offset statements were used and insert them. "
-                        "This is always safe to do for ren'py 8, but as it is based on a heuristic it can be disabled. "
-                        "The generated code is exactly equivalent, only slightly more cluttered.")
-
-    parser.add_argument('file', type=check_inpath, nargs='+',
-                        help="The filenames to decompile. "
-                        "All .rpyc files in any directories passed or their subdirectories will also be decompiled.")
-
-    parser.add_argument('--try-harder', dest="try_harder", action="store_true",
-                        help="Tries some workarounds against common obfuscation methods. This is a lot slower.")
+def check_inpath(inp, strict=True):
+    """Helper to check if given path exists and cast it to pathlikes."""
+    return pt(inp).resolve(strict)
 
-    parser.add_argument('--register-sl-displayable', dest="sl_custom_names", type=str, nargs='+',
-                        help="Accepts mapping separated by '=', "
-                        "where the first argument is the name of the user-defined displayable object, "
-                        "and the second argument is a string containing the name of the displayable,"
-                        "potentially followed by a '-', and the amount of children the displayable takes"
-                        "(valid options are '0', '1' or 'many', with 'many' being the default)")
 
-    args = parser.parse_args()
+def _parse_args():
+    """python3 unrpyc.py [-c] [-d] [--python-screens|--ast-screens|--no-screens] file [file ...]"""
+
+    _cc_num = set_cpu_num()
```

Comment:
Why all the leading underscores? I understand why _parse_args() got one, but there's generally no reason to use leading underscores for function locals according to PEP8.